src/AddressRegistry.Api.BackOffice.Handlers.Lambda/Handlers/SqsLambdaHandler.cs

```diff
@@ -132,5 +57,12 @@ private async Task ValidateIfMatchHeaderValue(TSqsLambdaRequest request, Cancell
             var streetNameHash = aggregate.GetAddressHash(addressPersistentLocalId);
             return streetNameHash;
         }
+        
+        protected override async Task HandleException<T>(TSqsLambdaRequest request, CancellationToken cancellationToken)
```

Comment:
I don't know if this is the right way to handle this. Since it's only used [here](https://github.com/Informatievlaanderen/basisregisters-sqs/blob/aa3e105895b63ef8f1a667b08546bfdc522a78ed/src/Be.Vlaanderen.Basisregisters.Sqs.Lambda/Handlers/SqsLambdaHandlerBase.cs#L65). Maybe just have a abstract method `HanldeAggregateIdIsNotFoundException` ? Or add a ticket error abstract property like `AggregateIdIsNotFoundTicketError`. Then the implementation can set the right `TicketError`.

Reply:
fixed